tests/torch/test_utils.py

```diff
@@ -76,10 +72,51 @@ def check_were_only_bn_training_state_changed(model: nn.Module, saved_state: _Mo
 
     runner = DataLoaderBNAdaptationRunner(model, "cuda")
 
-    randomly_change_model_state(model)
+    change_model_state(model)
```

Comment:
Do you mean just inside the test `test_bn_training_state_switcher` or also in `test_training_mode_switcher`? In case both has to be modified I can simply change the `change_model_state` function

Reply:
Please do it only in the `test_bn_training_state_switcher`, `test_training_mode_switcher` is ok